src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java

```diff
@@ -152,12 +154,26 @@ public boolean onStoreUnreachable(BackOffer backOffer) {
     return false;
   }
 
-  protected Kvrpcpb.Context makeContext(TiStoreType storeType) {
-    return region.getReplicaContext(java.util.Collections.emptySet(), storeType);
+  private Kvrpcpb.Context addTraceId(Kvrpcpb.Context context, SlowLog slowLog) {
+    long traceId = slowLog.getTraceId();
+    return Kvrpcpb.Context.newBuilder(context)
+        .setTraceContext(
+            Tracepb.TraceContext.newBuilder()
+                .setDurationThresholdMs(
+                    (int) (slowLog.getThresholdMS() / conf.getRawKVServerSlowLogFactor()))
```

Comment:
usually, a factor is used to be multiple with the base number.

Reply:
How about `rawkv_server_slowlog_threshold_divisor`?

Reply:
I prefer multiplying the factor.

Reply:
Why not just use a normal tracing threshold?

Reply:
> I prefer multiplying the factor.

The parameter is now multiplying the threshold.

Reply:
> Why not just use a normal tracing threshold?

We may want to set the threshold for tikv less than the client. For example half of the client threshold.

Reply:
it's better to limit the range of threshold factor to (0, 1) and multiple the factor in this place.